src/lib/modem.py

```diff
@@ -1,70 +1,155 @@
 import time
 from network import LTE
+from error_handling import ErrorHandler, COLOR_MODEM_FAIL
```

Comment:
there is no COLOR_MODEM_FAIL in error handling!

Reply:
... should work now with the last fixup!